packages/kolibri-common/components/SearchFiltersPanel/AccordionSelectGroup.vue

```diff
@@ -0,0 +1,468 @@
+          };
+        });
+      },
+      enabledAccessibilityOptions() {
+        return this.accessibilityOptionsList.filter(a => !a.disabled);
+      },
+      contentLevelsList() {
+        return this.availableGradeLevels.map(key => {
+          const value = ContentLevels[key];
+          let translationKey;
+          if (key === 'PROFESSIONAL') {
+            translationKey = 'specializedProfessionalTraining';
+          } else if (key === 'WORK_SKILLS') {
+            translationKey = 'allLevelsWorkSkills';
+          } else if (key === 'BASIC_SKILLS') {
+            translationKey = 'allLevelsBasicSkills';
+          } else {
+            translationKey = camelCase(key);
+          }
+          return {
+            value,
+            disabled: this.searchableLabels && !this.searchableLabels.grade_levels.includes(value),
+            label: this.coreString(translationKey),
+          };
+        });
+      },
+      enabledContentLevels() {
+        return this.contentLevelsList.filter(c => !c.disabled);
+      },
+      channelOptionsList() {
+        return this.availableChannels.map(channel => ({
+          value: channel.id,
+          disabled: this.searchableLabels && !this.searchableLabels.channels.includes(channel.id),
+          label: channel.name,
+        }));
+      },
+      enabledChannelOptions() {
+        return this.channelOptionsList.filter(c => !c.disabled);
+      },
+      langId() {
+        return Object.keys(this.value.languages)[0];
+      },
+      accessId() {
+        return Object.keys(this.value.accessibility_labels)[0];
+      },
+      selectedLanguage() {
+        if (!this.langId && this.enabledLanguageOptions.length === 1) {
+          return this.enabledLanguageOptions[0];
+        }
+        return this.languageOptionsList.find(o => o.value === this.langId) || {};
+      },
+      selectedAccessibilityFilter() {
+        if (!this.accessId && this.enabledAccessibilityOptions.length === 1) {
+          return this.enabledAccessibilityOptions[0];
+        }
+        return this.accessibilityOptionsList.find(o => o.value === this.accessId) || {};
+      },
+      levelId() {
+        return Object.keys(this.value.grade_levels)[0];
+      },
+      selectedLevel() {
+        if (!this.levelId && this.enabledContentLevels.length === 1) {
+          return this.enabledContentLevels[0];
+        }
+        return this.contentLevelsList.find(o => o.value === this.levelId) || {};
+      },
+      channelId() {
+        return Object.keys(this.value.channels)[0];
+      },
+      selectedChannel() {
+        if (!this.channelId && this.enabledChannelOptions.length === 1) {
+          return this.enabledChannelOptions[0];
+        }
+        return this.channelOptionsList.find(o => o.value === this.channelId) || {};
+      },
+    },
+    methods: {
+      isChecked(inputKey, value) {
+        return this.isSelected(inputKey, value) || this.isEnabledButNotSelected(inputKey, value);
```

Comment:
I think Marcella's comment is because we are considering this `isEnabledButNotSelected` value as correct value for `isChecked`, and I wonder, if its not selected, why would we check somthing if its not selected, even if its enabled?

Reply:
This just makes it so that you see the changes to what is selected even if it's not directly clicked by the user (ie, you select Spanish, then the other available filters are disabled/checked-automatically if there are no Spanish resources for them).

My latest updates make use of this (fixing the issue where everything is checked & disabled because I misread/understood the enabled computed properties).

However, I can probably just remove the `isEnabledButNotSelected` altogether if we handle selections like we do in Learn because part of the issue is to do with a reactivity issue somewhere if I recall correctly.